### ISSUE TITLE
Buy & register - sendy

### ISSUE THREAD

**Issue by cevap** [cevap/ion#43]
- Sendy url: https://sendy.co/ (one time $ 59)
 - Cross connected projects: https://github.com/cevap/ioncore.xyz/projects

**Comment by LordDarkHelmet** [comment]
It sounds like an interesting service. What are the downsides? What do services like Mail Chimp offer that is neglected by this service? As this is self hosted are we risking being flagged as a spamming site?

**Comment by cevap** [comment]
From current point of view, there is only one downside from point of view of this project:
 - $59 

(_It is not free and one of main goals was to use only free services. However, we had also to buy a domain_)

Pro side:
 - **It is already integrated to http://ioncore.xyz/**, which is hosted on GitHub (yes, for free)
 - **It is not a hosted service**
 - No additional server costs
 - No additional maintenance like most products offer
 - fully automated
 - Community is able to maintenance it (_it is very easy, [get started](https://sendy.co/get-started))

Not sure if pro/contra:
 - We can ask Amazon to limit the amount of send messages.

We have enough donations to buy this license now for $ 59, that is lifetime license for the domain. We should also take in ming, only one domain can be used and we might want to extend domain to 10 years, however, I hope to get it sponsored by ionomy or any other investors of ion coin.

What do you say? I would buy it immediately as then we can also use immeaditally our [announcement system](http://ioncore.xyz/en/list/announcements/join/). I am unsure, community is inactive and it would be waste of time to offer any kind of contacting or additional administrative tasks other than github, that's why I might disable [contact](http://ioncore.xyz/en/contact/) directly in homepage. However, all contact possibilities are there :). 

And yeah, should create another issue for that, I kept those emails:
```
For general enquiries and press: contact@ioncore.xyz (not for support).
To report security issues: security@ioncore.xyz (not for support).
```

I will keep them unless community/somebody else disagrees and shows me the point in doing it differently, what do you say?

**Comment by LordDarkHelmet** [comment]
I am not concerned with the price. It is extremely good given what is being offered. My remaining concern about being flagged as a spam site is still there. The resolution to this is most likely a series of internal restrictions to prevent unneeded updates or undesired emails from being sent.

**Comment by cevap** [comment]
You have always a possibility to become faked. I think we speak about things without some clarification.

 - How would we get flagged as spam?
 - How can our server be hacked/flagged?
   - We host currently on github
   - responsibility: github's server
   - liability: github's server

Let's get through this. In general channel on slack DK uses such service where I brought your point to discussion.
 - Email verification prevents non account holders to enter fake addresses
 - Nobody will receive any announcements if email has not been confirmed
 - Against bots some additional steps can be taken, however, it is not really part of this topic in my opinion, maybe sligthly.

I think, if we discuss about it, we should bring some points from here: http://media.amazonwebservices.com/AWS_Amazon_SES_Best_Practices.pdf

I also think, this is nice usecase of what happens even if you are blocked by mistake: https://sendy.co/forum/discussion/675/sendy-is-beautiful-amazon-ses-not-so-much-/p1

Most sophisticated spams are done by taking over someone's server, that is huge problem as server and it's software, HW, ... in combination. 

I know that ionomy uses mail chimp (or used, not sure anymore). Let's invite Adam to tell us about their experience, as it could be more worth then our discussion about it :).

**Comment by cevap** [comment]
@LordDarkHelmet , we bough it, you have access to it.

I do not mind testing/getting additional services, not only [mail chimp](https://mailchimp.com/), but maybe a german one too: [Newsletter2go](https://www.newsletter2go.de/).

**Comment by cevap** [comment]
I will close this ticket now as we do have bought it already. For further discussion I can reopen this link, however, lets better create then new project simply about that topic, something like

 - Project requirements for mass emails
 - Comparison of services according to our requirements
 - Make a decision which service is to be used

Until then, we can have more tests now with sendy, Adam tested also couple of them, I miss his input/suggestion on this topic as he used several different services.

@LordDarkHelmet if you can please create this project, if not, I'll do it anyway. I hope you did not miss credentials sent to you.
